src/settings/AddInboxRuleDialog.ts

```diff
@@ -107,10 +108,12 @@ export function show(mailBoxDetail: MailboxDetail, ruleOrTemplate: InboxRule) {
 }
 
 export function createInboxRuleTemplate(ruleType: string | null, value: string | null): InboxRule {
-	const template = createInboxRule()
-	template.type = ruleType || InboxRuleType.FROM_EQUALS
-	template.value = getCleanedValue(neverNull(ruleType), value || "")
-	return template
+	return createInboxRule({
+		type: ruleType || InboxRuleType.FROM_EQUALS,
+		value: getCleanedValue(neverNull(ruleType), value || ""),
+		// @ts-ignore
+		targetFolder: null, // FIXME
```

Comment:
FIXME

Reply:
Bunch of those "don't look at me, you were doing it before, I'm just pointing it out" fixmes here.